capivaraProjects/App.tsx

```diff
@@ -18,69 +18,42 @@ import {
   StatusBar,
 } from 'react-native';
 
-import {
-  Header,
-  LearnMoreLinks,
-  Colors,
-  DebugInstructions,
-  ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+
+import  Login  from './views/Login';
 
 declare const global: {HermesInternal: null | {}};
 
 const App = () => {
   return (
-    <>
-      <SafeAreaView>
-        <ScrollView
-          contentInsetAdjustmentBehavior="automatic"
-          style={styles.scrollView}>
-          <View style={styles.body}>
-            <Text style={styles.sectionTitle}> Hello World</Text>
-          </View>
-        </ScrollView>
-      </SafeAreaView>
-    </>
+      <View style={styles.container}>
+        <View style={styles.sectionViewTitle}>
+          <Text style={styles.sectionTextTitle}> Bem-vindo (a) á Taqtile!</Text>
```

Comment:
```suggestion
          <Text style={styles.sectionTextTitle}> Bem-vindo(a) à Taqtile!</Text>
```

Só dando um tapa no português ai 😆

Reply:
hahaha, não tinha percebido!